modules/accessing-windows-node-using-rdp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-windows-container-issues.adoc
+
+[id="accessing-windows-node-using-rdp_{context}"]
+= Accessing a Windows node using RDP
+
+You can access a Windows node by using a Remote Desktop Protocol (RDP).
+
+.Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You have created a Windows MachineSet.
+* You have added the key used in the `cloud-private-key` secret and the key used when creating the cluster to the ssh-agent. For security reasons, remember to remove the keys from the ssh-agent after use.
```

Comment:
@aravindhp I researched this briefly, and found that this mostly refers to an [internal Red Hat script](https://github.com/eparis/ssh-bastion). We don't have any official docs on the `ssh-bastion`, and this doesn't seem like something a regular admin would know about. The most official looking doc I found for configuring this was a Knowledge Base article [here](https://access.redhat.com/solutions/4073041). Since this relies on a script residing in GitHub, we likely won't be able to include this directly in the official docs. However, I could include the KB article I linked if this is an absolute requirement. Does the KB article seem like the best idea here? Or am I missing something?

Reply:
Yeah we can link to the KB article which is referring to the internal `ssh-bastion`. cc: @gmarkley-VI